packages/html/src/global_attributes.rs

```diff
@@ -25,2273 +71,2275 @@ macro_rules! trait_methods {
     // Don't rename the incoming ident
     ( $name:ident; ) => { (stringify!($name), None, false) };
 }
-pub trait GlobalAttributes {
-    trait_methods! {
-        @base
 
-        /// Prevent the default action for this element.
-        ///
-        /// For more information, see the MDN docs:
-        /// <https://developer.mozilla.org/en-US/docs/Web/API/Event/preventDefault>
-        prevent_default: "dioxus-prevent-default";
+trait_methods! {
```

Comment:
Can we keep it so we're running the impls inside the trait declared outside the macro?

This should keep the diff smaller and be a little bit more obvious from the outside what's happening.

Reply:
We could do that but then the map_global_attributes and map_svg_attributes functions would be associated with the trait

Reply:
Oh, I understand now. Sorry about that, I will merge an equivalent to master so the diff goes down.